build-system/tasks/bundle-size/index.js

```diff
@@ -53,23 +57,42 @@ const bundleSizeAppBaseUrl = 'https://amp-bundle-size-bot.appspot.com/v0/';
  * @return {Map<string, number>} the bundle size in KB rounded to 2 decimal
  *   points.
  */
-function getBrotliBundleSizes() {
+async function getBrotliBundleSizes() {
   // Brotli compressed size fluctuates because of changes in the RTV number, so
   // normalize this across pull requests by replacing that RTV with a constant.
   const bundleSizes = {};
 
   log(cyan('brotli'), 'bundle sizes are:');
-  for (const filePath of globby.sync(fileGlobs)) {
-    const normalizedFileContents = fs
-      .readFileSync(filePath, 'utf8')
-      .replace(new RegExp(internalRuntimeVersion, 'g'), normalizedRtvNumber);
-
-    const relativeFilePath = path.relative('.', filePath);
-    const bundleSize = parseFloat(
-      (brotliSize.sync(normalizedFileContents) / 1024).toFixed(2)
+  const values = report(process.cwd(), content =>
+    content.replace(
+      new RegExp(internalRuntimeVersion, 'g'),
+      normalizedRtvNumber
+    )
+  );
+  let next = await values.next();
+  let pathMap = undefined;
+  if (!next.done) {
```

Comment:
Nit: If we flip this to `if (next.done) return bundleSizes;`, then this `if` block goes away and `pathMap` can be `const`

Reply:
Good point, will address!